sdk/tools/cabman/CCFDATAStorage.cxx

```diff
@@ -0,0 +1,167 @@
+/*
+ * PROJECT:     ReactOS cabinet manager
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     CCFDATAStorage class implementation for Linux/Unix
+ * COPYRIGHT:   Copyright 2017 Casper S. Hornstrup (chorns@users.sourceforge.net)
+ *              Copyright 2017 Colin Finck <mail@colinfinck.de>
+ *              Copyright 2018 Dmitry Bagdanov <dimbo_job@mail.ru>   
+ */
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+
+#if !defined(_WIN32)
+#include <dirent.h>
+#endif
+
+#include "cabinet.h"
+#include "raw.h"
+#include "mszip.h"
+
+#if !defined(CAB_READ_ONLY)
+
+CCFDATAStorage::CCFDATAStorage()
+/*
+* FUNCTION: Default constructor
+*/
```

Comment:
If you want to use per-function comments, please stick to a format that is doxygen compatible, preferrably the one outlined here:
https://reactos.org/wiki/Coding_Style

Reply:
The whole CABMAN project uses that style of function comments, so decided to not change them to maintain consistency. 
Now the comments changed to the reactos' standard (only in file CCFDATAStorage.cxx file, though).
Commit fc996347e8